changelog.adoc

```diff
@@ -1,6 +1,10 @@
 = KafkaHealthCheck
 :icons: font
 
+== Version 1.1.0
+
+* Always set the kafka consumer group of this health check
```

Comment:
This changelog entry doesn't really match the change. How about something like "Make consumer groups unique by appending a random UUID when no group ID is configured explicitly.